app/src/main/res/values-fr/strings.xml

```diff
@@ -113,9 +116,13 @@
     <string name="category_basic_settings">Paramètres de base</string>
     <string name="title_resolution_list">Sélectionner la résolution et les FPS à atteindre</string>
```

Comment:
These needs to be updated to match the English changes

Reply:
Changed thank you